### PR TITLE
FLA-231 implements demo create account

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- FLA-231 - implements demo account

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Running the app locally

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/gettingStarted.md [hunk 1]
```diff
@@ -10,48 +10,11 @@ This guide will provide you with all information you need to get started.
...
-      "url": "string"
-    },
-    "cancel": {
-      "url": "string"
-    }
-  }
-}'
-
-```
+| 88da92db-0791-491e-8c58-1a969e67d2fb | Bob Ross, a user without CSO account but a bceid profile |
```
Comment (sdevalapurkar-bcgov) [inline docs/gettingStarted.md:17]:
```suggestion
| 88da92db-0791-491e-8c58-1a969e67d2fb | Bob Ross, a user without a CSO account but with a BCeID profile |
```

File: src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/SeedAccountTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    @BeforeAll
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+        Mockito.when(efilingAccountServiceMock.createAccount(Mockito.any())).thenReturn(AccountDetails.builder().universalId(UUID.randomUUID()).create());
+        sut = new SeedAccount(efilingAccountServiceMock);
+    }
+
+    @Test
+    @DisplayName("OK: when application start should seed 2 accounts")
```
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/SeedAccountTest.java:35]:
```suggestion
    @DisplayName("OK: on application start should seed 2 accounts")
```

File: src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/SeedAccountTest.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    public void whenInitShouldCreate2Accounts() {
```
Comment (sdevalapurkar-bcgov) [inline src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/SeedAccountTest.java:36]:
```suggestion
    public void onInitShouldCreate2Accounts() {
```

File: docs/gettingStarted.md (hunk 1 above)
Comment (alexjoybc) [reply docs/gettingStarted.md:17]: Hey @sdevalapurkar-bcgov the other guys are not Bob Ross, they are `temp` and `temp` I need some recommendations to give them the name they deserve, @adodge-bcg @akroon3r @TayGov @KFloodBCGov @karthikmsqa

File: src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        init();
+    }
+
+    private void init() {
+
+        CreateAccountRequest createAccount1 = CreateAccountRequest
+                .builder()
+                .universalId(Keys.ACCOUNT_WITH_EFILING_ROLE)
+                .firstName("temp")
```
Comment (alexjoybc) [inline src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java:35]: Temp??? No we can do better 🤣🤣🤣

File: docs/gettingStarted.md (hunk 1 above)
Comment (akroon3r) [reply docs/gettingStarted.md:17]: Ross had one son, Robert Stephen "Steve" Ross,[33] with his first wife, Vivian Ridge. Steve, also a talented painter, occasionally appeared on The Joy of Painting and became a Ross-certified instructor.[12] Steve appeared on camera in the last episode of Season 1, in which he read a series of general "how-to" questions sent in by viewers during the season, and Bob answered them one at a time, technique by technique, until he had completed an entire painting.

Ross and Ridge's marriage ended in divorce in 1977. Ross and his second wife, Jane, had no children together. In 1992, Jane died of cancer. In 1995, two months before his death, Ross married for a third time, to Lynda Brown.[34]

Ross was very secretive about his life and had a great liking for privacy, with only a tight circle of friends. Some of only a few interviews with his close-knit circle of friends and his family can be found in the 2011 PBS documentary Bob Ross: The Happy Painter.[11]

His company, Bob Ross Inc., is protective of his intellectual property and his privacy to this day.

File: src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java (hunk 4 above)
Comment (adodge-bcg) [reply src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java:35]: Im with Adam, one should be Bob Ross and the other should be Steve or Steven Ross

File: src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+        logger.info("Account 1 created {}", account1.getUniversalId());
+
+        CreateAccountRequest createAccount2 = CreateAccountRequest
+                .builder()
+                .universalId(Keys.ACCOUNT_WITHOUT_EFILING_ROLE)
+                .firstName("temp")
+                .lastName("temp")
+                .middleName("temp")
+                .email("temp")
```
Comment (adodge-bcg) [inline src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java:51]:
```suggestion
.firstName("Bob")
.lastName("Ross")
.middleName("Norman")
.email("bob.ross@example.com")
```

File: src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+    private void init() {
+
+        CreateAccountRequest createAccount1 = CreateAccountRequest
+                .builder()
+                .universalId(Keys.ACCOUNT_WITH_EFILING_ROLE)
+                .firstName("temp")
+                .lastName("temp")
+                .middleName("temp")
+                .email("temp")
```
Comment (adodge-bcg) [inline src/backend/libs/efiling-demo-starter/src/main/java/ca/bc/gov/open/jag/efiling/demo/SeedAccount.java:38]:
```suggestion
                .firstName("Steve")
                .lastName("Ross")
                .email("steven.ross@example.com")
```

File: docs/gettingStarted.md (hunk 1 above)
Comment (alexjoybc) [reply docs/gettingStarted.md]: As per this article I went with bob ross wives but scrambled their last names ;-), thanks @akroon3r for your deep analyse on bob

**Pull request merged**